READMEs/c2d-flow.md

```diff
@@ -90,26 +72,20 @@ DATASET_compute_values = {
 }
 
 # Create the Service
-from ocean_lib.services.service import Service
+from ocean_lib.structures.file_objects import UrlFile
 DATASET_compute_service = Service(
     service_id="2",
     service_type="compute",
     service_endpoint=ocean.config_dict["PROVIDER_URL"],
     datatoken=DATASET_datatoken.address,
-    files=DATASET_files,
+    files=[UrlFile(url=DATASET_url)],
```

Comment:
Cleaner: rather than rebuilding a UrlFile object from the raw DATASET_url, you should be able to retrieve the existing one from the asset's list of files. (And less risky: what if the asset object itself relies on referencing the in-memory objects?)

Reply:
Done